GVRf/Sample/gvr-sample/src/org/gearvrf/sample/SampleCubeScript.java

```diff
@@ -160,23 +153,20 @@ public void onInit(GVRContext gvrContext) {
     @Override
     public void onStep() {
         FPSCounter.tick();
-        mFrontBoardObject.getRenderData().getMaterial().setOpacity(1.0f);
-        mFrontBoardObject2.getRenderData().getMaterial().setOpacity(1.0f);
-        mFrontBoardObject3.getRenderData().getMaterial().setOpacity(1.0f);
+        mFrontFace.getRenderData().getMaterial().setOpacity(1.0f);
```

Comment:
Actually gvrsample app and the following part of the codes can explain why it continues doing it in onStep().

It does want to change the texture into half transparent when your eye point to it. And when your eyes' focus leave the texture, it will return back to opaque.